new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioTwoDomainsInImage.java

```diff
@@ -0,0 +1,367 @@
+  private final String domainUid1 = "istio-dii-wdt-1";
+  private final String domainUid2 = "istio-dii-wdt-2";
+  private final String adminServerPodName1 = domainUid1 + "-" + adminServerName;
+  private final String adminServerPodName2 = domainUid2 + "-" + adminServerName;
+
+  private static Map<String, Object> secretNameMap;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Creating unique namespace for Domain1");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace1 = namespaces.get(1);
+
+    logger.info("Creating unique namespace for Domain2");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    domainNamespace2 = namespaces.get(2);
+
+    // Label the domain/operator namespace with istio-injection=enabled
+    Map<String, String> labelMap = new HashMap();
+    labelMap.put("istio-injection", "enabled");
+
+    assertDoesNotThrow(() -> addLabelsToNamespace(domainNamespace1,labelMap));
+    assertDoesNotThrow(() -> addLabelsToNamespace(domainNamespace2,labelMap));
+    assertDoesNotThrow(() -> addLabelsToNamespace(opNamespace,labelMap));
+
+    logger.info("Namespaces [{0}, {1}, {2}] Labeled ",opNamespace, domainNamespace1, domainNamespace2);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace1,domainNamespace2);
+  }
+
+  /**
+   * Create two domains using domainhome-in-image model.
+   * Add istio configuration with default readinessPort. 
+   * Deploy istio gateway and virtual service on each domain namespaces.
+   * Add host information to gateway and virtual service configurations.
+   * Put the namespace.org as host configuration 
+   * Verify server pods are in ready state and services are created
+   * Verify login to WebLogic console on domain1 through istio ingress http 
+   * port by passing host information in HTTP header.
+   * Deploy a web application to domain1 through istio ingress http port 
+   * using host information in HTTP header. 
+   * Access web application through istio http ingress port using host 
+   * information in HTTP header.
+   * Repeat the same steps for domain2.
+   */
+  @Test
+  @DisplayName("Two WebLogic domainhome-in-image with single istio ingress")
+  @Slow
+  @MustNotRunInParallel
```

Comment:
These tests can be run in parallel, is there a reason why it shouldn't be?

Reply:
Remove the tag @MustNotRunInParallel